.github/workflows/downstream_tests.yml

```diff
@@ -493,3 +493,43 @@ jobs:
           touch tests/utils/__init__.py
           pytest tests
         timeout-minutes: 15
+  
+  darts:
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+        os: ["ubuntu-latest"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install uv
+        uses: astral-sh/setup-uv@v5
+        with:
+          enable-cache: "true"
+          cache-suffix: ${{ matrix.python-version }}
+          cache-dependency-glob: "pyproject.toml"
+      - name: clone-darts
+        run: |
+          git clone https://github.com/unit8co/darts.git --depth=1
+          cd darts
+          git log
+      - name: install-deps
+        run: |
+          cd darts
+          uv pip install -r requirements/core.txt -r requirements/dev.txt --system
+      - name: install-narwhals-dev
+        run: |
+          uv pip uninstall narwhals --system
+          # TODO(FBruzzesi): Install polars from darts requirements when they introcude it
+          uv pip install -e ".[polars]" --system
```

Comment:
We thought about not adding an extra "polars" to the darts package setup and let the user manually install it. Otherwise we'd also have to add pyarrow, ... extras.

Reply:
Thanks for the info @dennisbader
I will remove the TODO then, but I am just wondering, are you going to manually install in CI for testing?

Reply:
That was our plan, yes. We will add a `requirements/optional.txt` requirements file that we only install in one of our CI checks that is supposed to install "all" + optional dependencies. See here:

https://github.com/unit8co/darts/blob/393355275fd9d7a0a59678687f92b13d591aba47/.github/workflows/develop.yml#L70

Would you advice against that?

Reply:
That's perfectly fine, we can eventually use such file then - similar for how we handle it for [plotly in CI](https://github.com/narwhals-dev/narwhals/blob/main/.github/workflows/downstream_tests.yml#L309)